Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift

```diff
@@ -83,9 +84,7 @@ public struct VideoRendererView: UIViewRepresentable {
     }
 
     public func makeUIView(context: Context) -> VideoRenderer {
-        let view = showVideo
-            ? utils.videoRendererFactory.view(for: id, size: size)
-            : VideoRenderer()
+        let view = videoRendererPool.acquireRenderer(size: size)
```

Comment:
Interesting approach - so the id is no longer taking into consideration. We just say this view is taken now, right?

Reply:
Exactly. This is alright because our views are just observers on RTCVideoTrack instances that are built to deliver their frames to multiple observers. So we can get advantage of it simplify our lives by just saying, give a view i can render on and just make sure we free up resources when not in use(e.g. when views are being removed from the viewHierarchy)